### ISSUE TITLE
[comments] comment button requires text selection

### ISSUE THREAD

**Issue by zhenyasav** [dxos/dxos#6085]
it seems counterintuitive that we can't comment on a specific position - other tools don't require selections, specific positions are sufficient. Some tools automatically snap a selection around the nearest word, but I am not sure that's critical.

**Comment by ZaymonFC** [comment]
@wittjosiah, this directly conflicts with #7210?

**Comment by wittjosiah** [comment]
@ZaymonFC yeah #7210 is interim fix, not having the button disabled and snapping to the nearest word (a la google docs) would be an improvement (this issue) but we chose to prioritize resolution first
